CONTRIBUTING.rst

```diff
@@ -131,9 +131,33 @@ Contributing new architectures
 If you want to contribute a new model pleas read the pages on
 :ref:`architecture-life-cycle` and :ref:`adding-new-architecture`.
 
+Pull request workflow
+---------------------
+Once your changes are tested locally you are ready to submit your changes as a pull
+request.
+
+1. **Create the Pull Request**
+
+   - Push your changes to your fork of the repository.
+   - In a browser, navigate to the original repository and click on "New Pull Request".
+   - Select your branch and provide a clear description of your changes.
+
+2. **CI and Addressing Feedback**
+
+   - Ensure that all Continuous Integration (CI) checks pass successfully.
+   - Be open to feedback and make necessary changes to your code and try to respond to
+     comments and questions in a timely and friendly manner.
+   - **Before your PR can be merged, you have to trigger a manual GPU check
+     by commenting "cscs-ci run" in the PR. This is needed to ensure that the
+     GPU tests are run on the latest code.**
```

Comment:
Only maintainers of metatrain can do that, random external contributors can not. Might be worth to make it clear